test/libyul/yulOptimizerTests/ssaReverser/simple.yul

```diff
@@ -0,0 +1,14 @@
+{
+	let a := mload(1)
+    let a_1 := mload(0)
```

Comment:
I think we should have another test in this directory that shows what happens with multiple assignments.